XenAdmin/Wizards/PatchingWizard/PlanActions/PatchPrechecksOnMultipleHostsAction.cs

```diff
@@ -38,28 +38,28 @@
 using System.IO;
 using XenAdmin.Network;
 using XenAdmin.Diagnostics.Checks;
-using System.Diagnostics;
-
-namespace XenAdmin.Wizards.PatchingWizard.PlanActions
-{
-    class PatchPrechecksOnMultipleHostsInAPoolPlanAction : PlanActionWithSession
-    {
+using System.Diagnostics;
+
+namespace XenAdmin.Wizards.PatchingWizard.PlanActions
+{
+    class PatchPrechecksOnMultipleHostsInAPoolPlanAction : PlanActionWithSession
+    {
         private readonly XenServerPatch patch;
         private readonly List<PoolPatchMapping> mappings;
         private List<Host> hosts = null;
 
-        public PatchPrechecksOnMultipleHostsInAPoolPlanAction(IXenConnection connection, XenServerPatch patch, List<Host> hosts, List<PoolPatchMapping> mappings)
-            : base(connection, string.Format("Precheck for {0} in {1}...", patch.Name, connection.Name))
+        public PatchPrechecksOnMultipleHostsInAPoolPlanAction(IXenConnection connection, XenServerPatch patch, List<Host> hosts, List<PoolPatchMapping> mappings)
+            : base(connection, string.Format("Precheck for {0} in {1}...", patch.Name, connection.Name))
         {
             this.patch = patch;
-            this.hosts = hosts;
+            this.hosts = hosts;
             this.mappings = mappings;
-        }
-
-        protected override void RunWithSession(ref Session session)
+        }
+
+        protected override void RunWithSession(ref Session session)
         {
             var master = Helpers.GetMaster(Connection);
-            var mapping = mappings.Find(m => m.XenServerPatch.Equals(patch) && m.MasterHost.uuid == master.uuid);
+            var mapping = mappings.Find(m => m.XenServerPatch.Equals(patch) && m.MasterHost != null && master != null && m.MasterHost.uuid == master.uuid);

```

Comment:
The documentation for Find says it returns "The first element that matches the conditions defined by the specified predicate, if found; otherwise, the default value for type T". What is the default value for a PoolPatchMapping? Is it null?